demos/src/config.js

```diff
@@ -1,6 +1,5 @@
-'use strict';
```

Comment:
Yes, ESLint is complaining because we have in our default rules:
```
"ecmaFeatures": {
        "modules": true
    }
```
I believe Babel adds 'use strict' when transpiled, although it doesn't look like we're doing any transpiling in o-grid. Would this be handled by OBT or should I override the above rule?